docs/swagger.yaml

```diff
@@ -687,15 +689,23 @@ components:
         - amountExcludingTax
         - taxAmount
         - taxPercentage
-    PaymentSourceEnum:
-      type: string
-      title: PaymentSourceEnum
-      enum:
-        - card
-        - voucher
-        - cash
-        - giftcard
-        - wallet
+    PaymentSourceList:
```

Comment:
Maybe rename to "PaymentSources" - as it is technically not a list 😅

Reply:
Done :)